static/js/components/PersonalForm.js

```diff
@@ -39,6 +39,37 @@ export default class PersonalForm extends ProfileFormFields {
     ui:                     UIState,
   };
 
+  renderRomanizedFields = () => {
+    const { profile } =  this.props;
+    const english = /^[A-Za-z]*$/;
+    let firstName = _.get(profile, ["first_name"], "");
+    let lastName = _.get(profile, ["last_name"], "");
+    let rFirstName = _.get(profile, ["romanized_first_name"], "");
+    let rLastName = _.get(profile, ["romanized_last_name"], "");
+    const showRomanizedFields = (
+      !english.test(firstName) || !english.test(lastName) || rFirstName || rLastName
```

Comment:
My understanding was that `Müller` was acceptable for Latin characters, specifically the `ü` falls within the CP-1252 character set that the ticket specifies so it shouldn't be required to convert that to a regular `u`.

Maybe @pdpinch can clarify? Do we want the romanized names to only be in the range of `a-zA-Z`?